challenge-03/README.md

```diff
@@ -150,8 +177,23 @@ deve conter no retorno da frase acima é "metro" no lugar de "metros".
 método), que será concatenada com a frase de retorno, mostrando a resposta
 correta, de acordo com os dados inseridos no objeto.
 */
-?
+pessoa.apresentacao = function () {
+	var artigo = 'o';
+	if (pessoa.sexo === 'feminino') {
+		var artigo = 'a';
+	};
```

Comment:
-1
Você não deve usar ponto-e-vírgula após estrutura condicionais (como if, swtich), estruturas de repetição (while, for), e funções literais. Para todos os outros casos você usa ;)